client/components/seo/seo-site-search-preview.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes } from 'react';
+
+export const SeoSiteSearchPreview = React.createClass( {
+	shouldComponentUpdate( next ) {
```

Comment:
no aversion - just figured I'd make a simple one since it's easy.

updated to use `react-pure-render/mixin` in 1eebbaf